tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -53,33 +53,41 @@ function HeaderComp() {
                 </a>
               </li>
             </ul>
-            {user && user.email && (
-              <div className="usa-nav__secondary">
-                <ul className="usa-nav__secondary-links">
-                  <li className="usa-nav__secondary-item">
+            <div className="usa-nav__secondary">
+              <ul className="usa-nav__secondary-links">
+                <li
+                  className={`${
+                    user && user.email ? 'display-block' : 'display-none'
+                  } usa-nav__secondary-item`}
+                >
+                  {user && user.email && (
                     <a href="/">
                       <FontAwesomeIcon
                         className="margin-right-1"
                         icon={faUserCircle}
                       />
-                      {user.email}
-                    </a>
-                  </li>
-                  <li className="usa-nav__secondary-item">
-                    <a
-                      className="sign-out-link"
-                      href={`${process.env.REACT_APP_BACKEND_URL}/logout/oidc`}
-                    >
-                      <FontAwesomeIcon
-                        className="margin-right-1"
-                        icon={faSignOutAlt}
-                      />
-                      Sign Out
+                      {user && user.email}
                     </a>
-                  </li>
-                </ul>
-              </div>
-            )}
+                  )}
+                </li>
+                <li className="usa-nav__secondary-item">
+                  <a
+                    className="sign-out-link"
+                    href={
+                      user && user.email
+                        ? `${process.env.REACT_APP_BACKEND_URL}/logout/oidc`
+                        : `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
+                    }
+                  >
+                    <FontAwesomeIcon
+                      className="margin-right-1"
+                      icon={faSignOutAlt}
+                    />
+                    {user && user.email ? 'Sign Out' : 'Sign In'}
+                  </a>
+                </li>
```

Comment:
This is the `li` that will always be present. It will change whether it says `Sign In` or `Sign Out` depending on the authentication status.